docs/modules/ROOT/pages/building.adoc

```diff
@@ -35,31 +35,33 @@ $ helm upgrade secret-operator deploy/helm/secret-operator \
 
 == Nix
 
-To build and deploy to the active Kind cluster, run:
+To build and deploy to the active Kind cluster, run the following bash script:
 
 [source,console]
 ----
 $ echo Building with Nix
 # Ensure that all submodules are up-to-date
 $ git submodule update --recursive --init
-# Ensure that the Cargo.lock is up-to-date
-# This is not required if you use a tool that invokes Cargo regularly anyway, such as Rust-Analyzer
-$ cargo generate-lockfile
 # Use crate2nix (https://github.com/kolloch/crate2nix) to convert Cargo.lock into a Nix derivation
-$ nix run -f . crate2nix generate 
+$ nix run -f . crate2nix generate
 # Build the Docker images
+# A custom Docker repository can be specified by appending `--argstr dockerName my-custom-repository`
 $ nix build -f . docker
 # Load the images onto the Kind nodes
-# Nix does not use the Docker daemon, instead it builds individual layers, as well as a script (`result/load-image`) that combines them into a Docker image archive
-$ kind load image-archive <(./result/load-image)
+# Nix does not use the Docker daemon, instead it builds individual layers, as well as
+# a script (`result/load-image`) that combines them into a Docker image archive.
+# `load-image` can also be piped into `docker load` to prepare for pushing to a registry.
+$ ./result/load-image | kind load image-archive /dev/stdin
```

Comment:
This does not work on my machine:
```
$ ./result/load-image | kind load image-archive /dev/stdin --name stackable-data-platform
No 'fromImage' provided
Creating layer 1 from paths: ['/nix/store/ir0ipi4lf9gp90by9m2is1fw137i5zyz-libunistring-1.0']
Creating layer 2 from paths: ['/nix/store/890nx4dsf41xijgd8bdncxk56kv3lak4-libidn2-2.3.2']
ERROR: failed to load image: command "docker exec --privileged -i stackable-data-platform-worker3 ctr --namespace=k8s.io images import --digests --snapshotter=overlayfs -" failed with error:
exit status 1
Command Output: ctr: archive/tar: invalid tar header
Creating layer 3 from paths: ['/nix/store/mhhlymrg2m70r8h94cwhv2d7a0c8l7g6-glibc-2.34-210']
```

@maltesander suggested `kind load image-archive (./result/load-image | psub)` which works for me.

Reply:
Yep, same here, same error and the other command works. (This is for the listener-operator)

Reply:
@fhennig Can you check which docker image your kind is using for the k8s nodes?

Reply:
This seems to have been an issue with trying to deploy the same image to multi-node kind clusters. Updated the instructions.

`psub` sadly isn't suitable here since it's a fishism.